src/clj/rems/api/services/organizations.clj

```diff
@@ -57,19 +54,12 @@
      :errors [{:type :t.actions.errors/duplicate-id
                :organization/id (:organization/id org)}]}))
 
-(defn edit-organization! [userid org]
-  (organizations/update-organization!
-   (:organization/id org)
-   (fn [db-organization]
-     (let [organization-owners (set (map :userid (:organization/owners db-organization)))
-           organization-owner? (contains? organization-owners userid)
-           owner? (contains? context/*roles* :owner)]
-       (merge db-organization
-              (cond-> org
-                true (dissoc :organization/id)
-                (not (or organization-owner? owner?)) (dissoc :organization/owners))))))
+(defn edit-organization! [{:organization/keys [id] :as org}]
```

Comment:
We have tried to avoid destructuring by namespaced keys since it makes it difficult to grep for the uses. This is not always followed in the test utils but perhaps should.